### PR TITLE
[HOPSWORKS-722] Remove Horovod

### PR DESCRIPTION
**Pull request by robzor92**

### CODE REVIEW

**Comment by robzor92** [comment]
https://github.com/logicalclocks/hops-testing/pull/188

File: recipes/default.rb [hunk 1]
```diff
@@ -234,23 +232,6 @@
   end
 
 
-  #
-  # PATH needs /usr/local/bin/mpicxx for horovod
-  # horovod - https://github.com/uber/horovod/blob/master/docs/gpus.md
-  #
-  if node['tensorflow']['need_mpi'] == 1 && node['cuda']['accept_nvidia_download_terms'] == "true"
```
Comment (tkakantousis) [inline recipes/default.rb]: @robzor92 is the `need_mpi` used anywhere else? if not we can remove it from the attributes as well, what do you think?

Comment (robzor92) [reply recipes/default.rb]: Had a look at distributed pytorch and one of their APIs need MPI installed on the machine. So after I have had time to investigate how we would integrate dist pytorch we can remove it. The need_mpi is used when building TF with MPI support, which may not be necessary anymore.

Comment (tkakantousis) [reply recipes/default.rb]: we will keep it for now, as part of distributed pytorch effort

**Pull request merged**